helm/auto-deployment/configuration.md

```diff
@@ -0,0 +1,444 @@
+# Advanced Configuration
+
+## Client Libraries
+
+The following are the client libraries used in Bold BI by default.
+
+```console
+optionalLibs: 'mongodb,mysql,influxdb,snowflake,oracle,google,clickhouse'
+```
+
+Read the optional client library license agreement from the following link.
+
+[Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+
+If you wish to inlcude only specific client libraries note the optional client libraries from the above link as comma separated names in your values.yaml file.
+
+## Cluster Provider
+
+Bold BI supports Azure Kubernetes Service (AKS), Elastic Kubernetes Service (EKS) and Google Kubernetes Engine (GKE) you can enter the value based on your cluster provider.
+
+```console
+# your cluster provider name.The supported values are aks,gke and eks.

```

Comment:
your cluster provider name. The supported values are aks, gke, and eks.

Reply:
concern addressed